### PR TITLE
updated typo in Primitive Lists

### PR DESCRIPTION
**Pull request by Akash3121**
updated typo in Primitive Lists Example:38 from creCreate to Create.
Line No: 940.

### CODE REVIEW

**Comment by Akash3121** [comment]
Hello @nikhilnanivadekar, Thanks for the Inspections.
I'm new to this, I need help regarding eclipsefdn/eca (The author(s) of the pull request is not covered by necessary legal agreements) - errored.
https://accounts.eclipse.org/legal/eca/validation/118828
I've signed the ECA and Revalidated but still it's same.
Thanks and Regards,
Akash

**Comment by Akash3121** [comment]
Hello @donraab, Thank you for your suggestion to go for a PR to correct a typo, https://github.com/eclipse/eclipse-collections/issues/1383#issuecomment-1238915208.
I'm new to this, I need help regarding **eclipsefdn/eca** (The author(s) of the pull request is **not** covered by necessary legal agreements) - errored.
https://accounts.eclipse.org/legal/eca/validation/118828
I've signed the ECA and Revalidated but it's still same. Is there anything that I can do to solve that error.
Thanks and Regards,
Akash

File: docs/2-Collection_Containers.adoc [hunk 1]
```diff
@@ -937,7 +937,7 @@ They are named *{IntArrayList}*, *{FloatArrayList}* etc.
 Current JVMs use one byte per boolean in a *boolean[]* (instead of one bit per boolean).
 Thus the *BooleanArrayList* is backed by a *java.util.BitSet* as an optimization.
 
-.our ways to creCreate an *IntArrayList*, use one of the following:
+.our ways to Create an *IntArrayList*, use one of the following:
```
Comment (donraab) [inline docs/2-Collection_Containers.adoc:940]: should be a lowercase c

**Review by donraab — CHANGES_REQUESTED** [review]
Hi @Akash3121, thank you for the contribution! Looks like the ECA validation is passing now. One minor change requested. Make sure to squash any commits and rebase and then I can approve once everything looks good.

File: docs/2-Collection_Containers.adoc (hunk 1 above)
Comment (Akash3121) [reply docs/2-Collection_Containers.adoc]: Yes Sir, will do that.

**Comment by Akash3121** [comment]
Hi @donraab, Thank you for guiding me in my initial contribution.
"C" changed to small case and squashed commits.
Thanks & Regards,
Akash.

**Review by donraab — APPROVED** [review]
LGTM! Thanks for the contribution!

**Pull request merged**